pkg/cable/libreswan/libreswan.go

```diff
@@ -212,7 +212,7 @@ func retrieveActiveConnectionStats() (map[string]int, map[string]int, error) {
 		}
 	}
 
-	return activeConnectionsRx, activeConnectionsTx, nil
+	return activeConnectionsRx, activeConnectionsTx, errors.Wrap(cmd.Wait(), "error waiting for whack to complete")
```

Comment:
I was wondering if we have to Wait even in the whack() method, but we use Run() method which seems to wait internally. Thanks for fixing @skitt